etc/make_release.py

```diff
@@ -621,7 +621,7 @@ def read_github_creds(github_token_file):
     Read the GitHub token from the specified file and return it as a string.
     """
 
-    token_re = re.compile('^(?:Token - )?(?P<tok>\w{40}).*$')
+    token_re = re.compile('^(?:Token - )?(?P<tok>\\w{40}).*$')
```

Comment:
```suggestion
    token_re = re.compile(r'^(?:Token - )?(?P<tok>\w{40}).*$')
```

Consider using a raw string to treat the backslash as a literal character. Raw strings appear frequently in https://docs.python.org/3/library/re.html.